packages/react-forms/src/InputDate/index.js

```diff
@@ -36,126 +36,134 @@ type State = {
   isOpen: boolean,
 };
 
-export default class InputDate extends React.Component<Props, State> {
-  component = React.createRef /*:: <HTMLElement> */();
-  refA = React.createRef /*:: <HTMLElement> */();
-  refB = React.createRef /*:: <HTMLElement> */();
-
-  state = {
-    current: undefined,
-    isOpen: false,
-  };
-
-  get isOpen(): boolean {
-    return Boolean(this.props.onToggle ? this.props.isOpen : this.state.isOpen);
-  }
-
-  setIsOpen = (isOpen: boolean) =>
-    this.props.onToggle
-      ? this.props.onToggle(isOpen)
-      : this.setState({ isOpen });
-
-  handleOpen = () => this.isOpen === false && this.setIsOpen(true);
-
-  handleClose = /* istanbul ignore next */ () =>
-    void (this.isOpen === true && this.setIsOpen(false));
-
-  handleCloseOnBlur = (evt: MouseEvent) => {
-    const { relatedTarget } = evt;
-    // istanbul ignore next (this is too DOM related)
-    if (
-      this.component.current && relatedTarget instanceof HTMLElement
-        ? !this.component.current.contains(relatedTarget)
-        : true
-    ) {
-      this.setIsOpen(false);
+const InputDate = ({
+  value,
+  onChange, // eslint-disable-line no-unused-vars
+  isOpen: isOpenProp, // eslint-disable-line no-unused-vars
+  onToggle, // eslint-disable-line no-unused-vars
+  min,
+  max,
+  ...props
+}: Props) => {
+  const component = React.createRef();
```

Comment:
We should either use `React.*` or `import { * } from 'react'`, but not mix them

Reply:
Sure thing.